app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,8 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".activity.ParkingStayActivity">
+        </activity>
```

Comment:
if you will let the body empty just finish the line above with />

Reply:
done